descriptors/ModuleDescriptor-template.json

```diff
@@ -47,7 +47,10 @@
           "methods": ["PUT"],
           "pathPattern": "/invoice/invoices/{id}",
           "permissionsRequired": ["invoice.invoices.item.put"],
-          "permissionsDesired": ["invoices.acquisitions-units-assignments.manage"],
+          "permissionsDesired": [
+            "invoices.acquisitions-units-assignments.manage",
+            "invoices.acquisitions-units-assignments.assign"
```

Comment:
Currently there is no logic implemented for checking "assign" permission. Please create a unit test toghether with this scenario implementation.

Reply:
`assign` acq units permission is part of POST and invoice creation, when we edit we manage acq units

BTW how is it connected to preventing approve/pay?

Reply:
to prevent you need totally new permission and logic in controller